### ISSUE TITLE
finetune.py 改变结果

### ISSUE THREAD

**Issue by Liampour** [tengyu1998/SCI#50]
你好
祝贺你完成了出色的工作。
我尝试使用提供的微调模型，但不确定如何使用其结果。作为数据，我从 ./data/finetune 文件夹和difficult.pt 模型输入了 25 张图像。我使用 25 张图像的数据集进行训练和测试（finetune.py 中的第 65、67 行）。以下是 finetune.py 的参数：

`--batch_size', type=int, default=1, help='batch size
--steps', type=float, default=100, help='finetune steps
--cuda', default=True, type=bool, help='Use CUDA to train model
--gpu', type=str, default='0', help='gpu device id
--lr', type=float, default=0.0005, help='learning rate
--seed', type=int, default=2, help='random seed
--save', type=str, default='results/finetune/', help='location of the data corpus
--model', type=str, default='./weights/difficult.pt', help='location of the data corpus` 

结果出现了一些图像，但difficult.pt 模型的行为似乎没有改变。
1) finetune.py 会改变 .pt 模型吗？
2) 我使用相同的数据进行训练和测试是否会影响 finetune.py 结果？
3) 我应该如何使用它的结果？

**Comment by tengyu1998** [comment]
您好，邮件已收到，我将尽快处理。
